erts/doc/guides/erl_dist_protocol.md

```diff
@@ -57,7 +57,43 @@ The integers in all multibyte fields are in big-endian order.
 
 The requests served by the EPMD are summarized in the following figure.
 
-![Summary of EPMD Requests](assets/erl_ext_fig.gif "Summary of EPMD Requests")
+```mermaid
+---
+title: Summary of EPMD Requests
+---
+
+sequenceDiagram
+    participant client as Client (or Node)
+    participant EPMD
+
+    Note over EPMD: Register a Node
+    client ->> EPMD: ALIVE2_REQ
+    EPMD -->> client: ALIVE2_X_RESP
+
+    Note over EPMD: Unregister a Node
+    client ->> EPMD: ALIVE_CLOSE_REQ
+
+    Note over client: Get the Distribution Port of Another Node
```

Comment:
I wonder if there is any better way to display these notes. Maybe it is better to split into multiple diagrams? Or convert this summary to a billet list and move the diagrams into each respective section?

Reply:
I was using the notes to segregate the different requests. A bullet list, and separate diagrams sound good to me. I feel that this diagrams works well as an overview or summary diagram as originally intended. So what I would do is keep the diagram, and add a bullet list, as well as separate diagrams.